res/gamedata/scripts/ui_mm_opt_gameplay.script

```diff
@@ -35,7 +35,8 @@ function opt_gameplay:InitControls(x, y, xml, handler)
 	
 	_st = xml:InitStatic("tab_gameplay:templ_item", nil)
 	xml:InitStatic("tab_gameplay:cap_fov", _st)
-	xml:InitTrackBar("tab_gameplay:track_fov", _st)
+	self.fov_track = xml:InitTrackBar("tab_gameplay:track_fov", _st)
+	self.fov_track_val = xml:InitStatic("tab_gameplay:cap_fov_val", _st)
```

Comment:
Hmm... maybe it will be better to implement an option for displaying current value of the track directly in the engine? It could be useful for reusability in the future.

Reply:
It looks like option just for option. If we'll need this in future, we'll make it in future. Now it'll be unused/dead code, which is not very good.

Reply:
@FreeZoneMods is right. It will be very useful to implement such option for trackbar in the engine so that other modders can use it.

Reply:
@ShokerStlk could you do that?

Reply:
UITrackBar.cpp does not support any text by default (it's not even child of the CUIStatic). I started doing similar to what GSC have in UIComboBox, but soon dropped it, and moved to Lua. To be honest i don't want to spend more time on this.

Reply:
Ok, i will try now to implenet this through engine.